lib/moab/file_group.rb

```diff
@@ -222,9 +222,8 @@ def harvest_directory(path, recursive, validated=nil)
     # @param validated [Boolean] if true, path is verified to be descendant of (#base_directory)
```

Comment:
might want to change the param name here too and say something about it not actually being used in this method?  is the param there just to maintain compatibility with callers that call similar methods and always pass that param in?  it does appear from a cursory grep of the codebase like at least one caller expects to pass that param.